Dockerfile

```diff
@@ -1,15 +1,16 @@
-FROM node:alpine
+FROM node:10-alpine
 
 WORKDIR /app
 
-RUN apk update && apk add python g++ make openssh git bash pdftk
-RUN export PYTHONPATH=${PYTHONPATH}:/usr/lib/python2.7
+RUN apk add --no-cache python make g++
 
-COPY ./package.json yarn.lock ./
+COPY ./scripts ./scripts
```

Comment:
```suggestion
COPY ./scripts scripts
```

Reply:
I know it's working the  way it is. If there are reasons to change this I would suggest to change it aligned with the [manual](https://docs.docker.com/v17.09/engine/userguide/eng-image/dockerfile_best-practices/#add-or-copy) which then would copy to absolute paths e.g.
```
COPY ./scripts /scripts/
```

Reply:
> I know it's working the way it is. If there are reasons to change this I would suggest to change it aligned with the [manual](https://docs.docker.com/v17.09/engine/userguide/eng-image/dockerfile_best-practices/#add-or-copy) which then would copy to absolute paths e.g.
> 
> ```
> COPY ./scripts /scripts/
> ```

I made these suggestins according to the same manual (copying to the WORKDIR)